FAULT TYPE: OPENSTACK_THROW_EXCEPTION-INSTANCE
TARGET COMPONENT: /nova/compute/manager.py
TARGET CLASS: ComputeManager
TARGET FUNCTION DEF: _shutdown_instance+2281;12
FAULT POINT: self._notify_about_instance_usage(context, instance, 'shutdown.start')
